src/rpc/misc.cpp

```diff
@@ -593,11 +625,11 @@ static const CRPCCommand commands[] =
     { "util",               "createblindedaddress",   &createblindedaddress,   true,  {} },
     { "util",               "verifymessage",          &verifymessage,          true,  {"address","signature","message"} },
     { "util",               "signmessagewithprivkey", &signmessagewithprivkey, true,  {"privkey","message"} },
-
+    { "util",               "tweakfedpegscript",      &tweakfedpegscript,      true,  {"claim_script"} },
     /* Not shown in help */
     { "hidden",             "setmocktime",            &setmocktime,            true,  {"timestamp"}},
     { "hidden",             "echo",                   &echo,                   true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
-    { "hidden",             "echojson",               &echo,                  true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
+    { "hidden",             "echojson",               &echo,                   true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
```

Comment:
unrelated space

Reply:
Re-added the line that was removed: The space change was just to fix the column alignment in this line.